std/traits.d

```diff
@@ -279,21 +279,21 @@ private
 
 package
 {
-    /// Add specific qualifier to the given type T.
+    /// Add the mutable qualifier to the given type T.
     template MutableOf(T)     { alias MutableOf     =              T  ; }
```

Comment:
This is `package` and shouldn't be displayed on the docs ... 

DScanner issue: https://github.com/dlang-community/D-Scanner/issues/493